tubearchivist/home/urls.py

```diff
@@ -3,18 +3,30 @@
 from django.conf import settings
 from django.contrib.auth.decorators import login_required
 from django.contrib.auth.views import LogoutView
+from django.shortcuts import redirect
 from django.urls import path
 from home import views
 
-urlpatterns = [
-    path("", login_required(views.HomeView.as_view()), name="home"),
-    path("login/", views.LoginView.as_view(), name="login"),
-    path(
+if settings.TA_AUTH_PROXY_LOGOUT_URL:
```

Comment:
This will raise an `AttributeError` if you don't configure that. You'll have to check that attribute first before you can access it, e.g. something like this should work:
```python
hasattr(settings, "TA_AUTH_PROXY_LOGOUT_URL")
```
That will return `True` if the attribute is set in `settings.py`, `False` otherwise.